Doc/using/cmdline.rst

```diff
@@ -994,3 +994,101 @@ Debug-mode variables
    Need Python configured with the :option:`--with-trace-refs` build option.
 
    .. versionadded:: 3.11
+
+.. _using-on-finding-modules:
+
+Finding modules
+---------------
+
+A module search path is initialized when Python starts. This module search path
+may be accessed at :data:`sys.path`.
+
+The first entry in the module search path is the directory that contains the
+input script, if there is one. Otherwise, the first entry is the current
+directory, which is the case when executing the interactive shell, a ``-c``
+command, or ``-m`` module.
+
+The :envvar:`PYTHONPATH` environment variable is often used to add directories
+to the search path. If this environment variable is found then the contents are
+added to the module search path.
+
+The next items added are the directories containing standard Python modules as
+well as any shared libraries (DLL/dylib/so) that these modules depend on. The
+directory with the platform-independent Python modules is called ``prefix``.
+The directory with the shared libraries is called ``exec_prefix``.
+
+The :envvar:`PYTHONHOME` environment variable may be used to set the ``prefix``
+and ``exec_prefix`` locations. Otherwise these directories are found by using
```

Comment:
Ill-advised setting of PYTHONPATH seems to be one of the big footshoots for novice users. Often there are recommendations to add setting PYTHONPATH in one's shell profile files or the like and then it is forgotten about until it comes back to haunt you when you upgrade to a new Python version or use a venv. I think we should emphasize somewhere in the section that: (1) the values of these variables apply to *all* versions of Python that you might be using and (2) that, in most cases, you should not need to set them.

Reply:
I agree, I have added a note to highlight this.